Persistent seeder crashes

See logs here: https://dashboard.heroku.com/apps/persistent-seeder-production/logs

```
2020-06-16T17:11:37.738905+00:00 app[persistent-seeder.1]:     at Module.load (internal/modules/cjs/loader.js:653:32)
2020-06-16T17:11:37.738905+00:00 app[persistent-seeder.1]:     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
2020-06-16T17:11:37.738905+00:00 app[persistent-seeder.1]:     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
2020-06-16T17:11:37.766909+00:00 app[persistent-seeder.1]: error Command failed with exit code 1.
2020-06-16T17:11:37.767100+00:00 app[persistent-seeder.1]: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2020-06-16T17:11:37.830877+00:00 heroku[persistent-seeder.1]: Process exited with status 1
2020-06-16T17:11:37.874364+00:00 heroku[persistent-seeder.1]: State changed from up to crashed
2020-06-16T17:11:53.825607+00:00 heroku[persistent-seeder.1]: State changed from crashed to starting
2020-06-16T17:13:08.903517+00:00 heroku[persistent-seeder.1]: Starting process with command `/entrypoint.sh`
2020-06-16T17:13:09.597409+00:00 heroku[persistent-seeder.1]: State changed from starting to up
2020-06-16T17:13:13.203232+00:00 app[persistent-seeder.1]: yarn run v1.17.3
2020-06-16T17:13:13.433730+00:00 app[persistent-seeder.1]: $ SC_ENV=persistent-seeder ts-node ./persistent-seeder.ts
2020-06-16T17:13:20.270114+00:00 app[persistent-seeder.1]: 
2020-06-16T17:13:20.272110+00:00 app[persistent-seeder.1]: /statechannels/monorepo/node_modules/ts-node/src/index.ts:423
2020-06-16T17:13:20.272111+00:00 app[persistent-seeder.1]:     return new TSError(diagnosticText, diagnosticCodes)
2020-06-16T17:13:20.272111+00:00 app[persistent-seeder.1]:            ^
2020-06-16T17:13:20.272370+00:00 app[persistent-seeder.1]: TSError: ⨯ Unable to compile TypeScript:
2020-06-16T17:13:20.272372+00:00 app[persistent-seeder.1]: persistent-seeder.ts(17,59): error TS2345: Argument of type '"./nitro-protocol.pdf"' is not assignable to parameter of type 'Partial<UploadFileOptions> | undefined'.
2020-06-16T17:13:20.272372+00:00 app[persistent-seeder.1]: 
2020-06-16T17:13:20.272373+00:00 app[persistent-seeder.1]:     at createTSError (/statechannels/monorepo/node_modules/ts-node/src/index.ts:423:12)
2020-06-16T17:13:20.272374+00:00 app[persistent-seeder.1]:     at reportTSError (/statechannels/monorepo/node_modules/ts-node/src/index.ts:427:19)
2020-06-16T17:13:20.272375+00:00 app[persistent-seeder.1]:     at getOutput (/statechannels/monorepo/node_modules/ts-node/src/index.ts:554:36)
2020-06-16T17:13:20.272375+00:00 app[persistent-seeder.1]:     at Object.compile (/statechannels/monorepo/node_modules/ts-node/src/index.ts:760:32)
2020-06-16T17:13:20.272375+00:00 app[persistent-seeder.1]:     at Module.m._compile (/statechannels/monorepo/node_modules/ts-node/src/index.ts:839:43)
2020-06-16T17:13:20.272377+00:00 app[persistent-seeder.1]:     at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
2020-06-16T17:13:20.272378+00:00 app[persistent-seeder.1]:     at Object.require.extensions.(anonymous function) [as .ts] (/statechannels/monorepo/node_modules/ts-node/src/index.ts:842:12)
2020-06-16T17:13:20.272378+00:00 app[persistent-seeder.1]:     at Module.load (internal/modules/cjs/loader.js:653:32)
2020-06-16T17:13:20.272378+00:00 app[persistent-seeder.1]:     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
2020-06-16T17:13:20.272379+00:00 app[persistent-seeder.1]:     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
2020-06-16T17:13:20.318965+00:00 app[persistent-seeder.1]: error Command failed with exit code 1.
2020-06-16T17:13:20.319316+00:00 app[persistent-seeder.1]: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2020-06-16T17:13:20.391752+00:00 heroku[persistent-seeder.1]: Process exited with status 1
2020-06-16T17:13:20.457353+00:00 heroku[persistent-seeder.1]: State changed from up to crashed
```

--- a/packages/persistent-seeder/persistent-seeder.ts
+++ b/packages/persistent-seeder/persistent-seeder.ts
@@ -14,7 +14,7 @@ export async function persistentSeeder(): Promise<void> {
   await web3tTabA.goto(WEB3TORRENT_URL + '/upload', {waitUntil: 'load'});
   await web3tTabA.bringToFront();
 
-  const url = await uploadFile(web3tTabA, true, metamask, './nitro-protocol.pdf');
+  const url = await uploadFile(web3tTabA, true, metamask, {filePath: './nitro-protocol.pdf'});
   console.log(`seeding: go to ${url}`);
 }
 

